bin/Execute_report.R

```diff
@@ -4,35 +4,38 @@ library(rmarkdown)
 
     make_option(c("-y", "--input_type"), type="character", default="featurecounts", help="Which type of input data is provided; must be one of [featurecounts, rsem, salmon]", metavar="character"),
-    make_option(c("-c", "--gene_counts"), type="character", default=NULL, help="path to raw count table or directory", metavar="character"),
-    make_option(c("-m", "--metadata"), type="character", default=NULL, help="path to metadata table", metavar="character"),
-    make_option(c("-d", "--model"), type="character", default=NULL, help="path to linear model design file", metavar="character"),
-    make_option(c("-t", "--gtf"), type="character", default=NULL, help="path to gtf table if using salmon/rsem input", metavar="character"),
-
-    make_option(c("-x", "--contrast_matrix"), type="character", default=NULL, help="path to contrast matrix file", metavar="character"),
-    make_option(c("-k", "--contrast_list"), type="character", default=NULL, help="path to contrast list file", metavar="character"),
-    make_option(c("-p", "--contrast_pairs"), type="character", default=NULL, help="path to contrast pairs file", metavar="character"),
-    make_option(c("-l", "--genelist"), type="character", default=NULL, help="path to gene list file", metavar="character"),
-    make_option(c("-e", "--relevel"), type="character", default=NULL, help="path to factor relevel file", metavar="character"),
+    make_option(c("-c", "--gene_counts"), type="character", default=NULL, help="Path to raw count table or directory", metavar="character"),
+    make_option(c("-m", "--metadata"), type="character", default=NULL, help="Path to metadata table", metavar="character"),
+    make_option(c("-d", "--model"), type="character", default=NULL, help="Path to linear model design file", metavar="character"),
+    make_option(c("-t", "--gtf"), type="character", default=NULL, help="Path to gtf table if using salmon/rsem input", metavar="character"),
+
+    make_option(c("-x", "--contrast_matrix"), type="character", default=NULL, help="Path to contrast matrix file", metavar="character"),
+    make_option(c("-k", "--contrast_list"), type="character", default=NULL, help="Path to contrast list file", metavar="character"),
+    make_option(c("-p", "--contrast_pairs"), type="character", default=NULL, help="Path to contrast pairs file", metavar="character"),
+    make_option(c("-l", "--genelist"), type="character", default=NULL, help="Path to gene list file", metavar="character"),
+    make_option(c("-e", "--relevel"), type="character", default=NULL, help="Path to factor relevel file", metavar="character"),
     make_option(c("-b", "--batch_effect"), action="store_true", default=FALSE, help="Batch effect correction."),
     make_option(c("-f", "--log_FC_threshold"), type="double", default=NULL, help="Log Fold Change threshold to consider a gene DE."),
+    make_option("--round_DE", type="integer", default=NULL, help="How many decimals to keep after rounding the DE analysis values."),
```

Comment:
In the `nextflow.schema` it's defined as string

Reply:
yep, just changed this, see above